katrain/popups.kv

```diff
@@ -753,14 +753,17 @@
             LabelledCheckBox:
                 id: fast
                 size_hint: 0.05,1
+                input_property: "general/load_fast_analysis"
+                on_press: root.update_config(True)
             DescriptionLabel:
                 text: i18n._("load sgf rewind")
                 halign: 'right'
                 size_hint: 0.45,1
             LabelledCheckBox:
                 id: rewind
-                active: True
                 size_hint: 0.05,1
+                input_property: "general/load_sgf_rewind"
```

Comment:
config.json should also have these fields

Reply:
not needed in the config.json.
the checkbox has default value.

when user change the check state,the field will be write back into the config file.

Reply:
anyway ,added in the config file.